### ISSUE TITLE
[COGNITION]: Buttons - All buttons SHOULD have a darker shade on hover and focus

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#9087]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️. Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Our `button[class="va-button-primary"]` becomes a darker green shade on `:hover` and `:focus` pseudo-states. Our regular blue button and secondary outlined button do not become a darker shade on `:focus`. This could be improved for users who rely on the keyboard to navigate or use those color change cues for identifying the element with focus. Screenshots attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Blue button has the same background color on hover and focus
- [ ] Blue button retains the yellow halo on focus
- [ ] Outlined secondary button has the same darker border and text on hover and focus
- [ ] Outlined secondary button retains the yellow halo on focus

## WCAG or Vendor Guidance (optional)
* [Focus Visible: Understanding SC 2.4.7](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html)

## Screenshots or Trace Logs

![Screen Shot 2020-05-13 at 4 57 11 PM](https://user-images.githubusercontent.com/934879/81871145-d73e9280-953c-11ea-8de6-204432298f1f.png)

---
![Screen Shot 2020-05-13 at 4 57 19 PM](https://user-images.githubusercontent.com/934879/81871155-dd347380-953c-11ea-83aa-adb123e98f57.png)

---
![Screen Shot 2020-05-13 at 4 57 40 PM](https://user-images.githubusercontent.com/934879/81871170-e3c2eb00-953c-11ea-94f0-9453723ad26e.png)

**Comment by 1Copenut** [comment]
Issue moved to [department-of-veterans-affairs/vets-design-system-documentation #235](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/235) via [**ZenHub**](https://www.zenhub.com/)
